packages/web/src/components/stat-banner/StatBanner.tsx

```diff
@@ -146,13 +153,17 @@ export const StatBanner = (props: StatsBannerProps) => {
                     onClick: onShare!,
                     icon: <IconShare />
                   },
-                  {
-                    text: messages.unblockMessages,
-                    onClick: () => {
-                      // TODO
-                    },
-                    icon: <IconUnblockMessages />
-                  }
+                  isBlocked
+                    ? {
+                        text: messages.unblockMessages,
+                        onClick: onUnblock!,
```

Comment:
do we need `!`? Can the prop be not optional?

Reply:
StatBanner can be empty, eg on track pages